src/main/resources/static/app/components/sidebar/lab-requests-sidebar-template.html

```diff
@@ -3,30 +3,46 @@
 This file is distributed under the GNU Affero General Public License
 (see accompanying file LICENSE).
 -->
+<script>
```

Comment:
I don't think so, I couldn't figure out how and we do it the same way on the requests sidebar page with the function resetFilter.